package.json

```diff
@@ -26,7 +26,9 @@
     "npm-run-all": "^4.1.2",
     "rollup": "0.57.1",
     "rollup-plugin-buble": "0.19.2",
+    "rollup-plugin-commonjs": "^9.1.3",
     "rollup-plugin-json": "2.3.0",
+    "rollup-plugin-node-resolve": "^3.3.0",
```

Comment:
Change to exact version? Perhaps add `save-exact=true` to a .npmrc file so we don't have to remember to set it ourselves?